spec/facility_factory_spec.rb

```diff
@@ -0,0 +1,64 @@
+require 'spec_helper'
+
+RSpec.configure do |config|
+config.formatter = :documentation
+end
+
+RSpec.describe FacilityFactory do
+
+  it "can initialize" do
+    expect(facility_factory = FacilityFactory.new).to be_an_instance_of(FacilityFactory)
+  end
+
+  #DmvDataService.new.co_dmv_office_locations
+
+  it 'can create a facility' do
+    facility_factory = FacilityFactory.new
+    office_locations = [{
+      :the_geom=>{:type=>"Point", :coordinates=>[-104.97443112500002, 39.75525297420336]},
+      :dmv_id=>"1",
+      :dmv_office=>"DMV Tremont Branch",
+      :address_li=>"2855 Tremont Place",
+      :address__1=>"Suite 118",
+      :city=>"Denver",
+      :state=>"CO",
+      :zip=>"80205",
+      :phone=>"(720) 865-4600",
+      :hours=>"Mon, Tue, Thur, Fri  8:00 a.m.- 4:30 p.m. / Wed 8:30 a.m.-4:30 p.m.",
+      :services_p=>"vehicle titles, registration, renewals;  VIN inspections",
+      :parking_no=>"parking available in the lot at the back of the bldg (Glenarm Street)",
+      :photo=>"images/Tremont.jpg",
+      :address_id=>"175164",
+      :":@computed_region_nku6_53ud"=>"1444"
+      }]
+    
```

Comment:
There are a couple different ways to handle these tests. 
1) what you're doing here is defining the test data in this office_locations array. You are defining the test data and the benefit of this is that you have total control over the data and you know it won't change. You only have one facility object though here, so you'd maybe want to add another to ensure that your code can create more than one facility. That can get long so sometimes we pull this test data into other files to read from (more on that later)

2) Another option is to call the DmvDataService.new.co_dmv_office_locations and use that instead of office_locations as the result that you're going to call assertions on. The benefit is that you have a very realistic data set (i.e. many facilities listed in an array), but the downside is that this data could change so you wouldn't always want to make assertions on the actual values. Instead, you might want to verify the structure (that certain keys exist) and check the data types of the values rather than the actual values. If you prefer being able to test the actual values, then going with option 1 would be best. But, generally testing that office attributes are not nil and the right datatype can be sufficient. 

Which option you choose will definitely depend on lots of factors, but thought I would share this context since it seems you were on the track to using one and then switched to the other!